bigtable/admin.go

```diff
@@ -671,6 +671,18 @@ func storageTypeFromProto(st btapb.StorageType) StorageType {
 	return SSD
 }
 
+// InstanceState is the state of the instance. This is output-only.
+type InstanceState int32
+
+const (
+	// NOT_KNOWN is state of the instance could not be determined
```

Comment:
nit: add a period to all of these. https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
```suggestion
	// NOT_KNOWN represents the state of an instance that could not be determined.
```